tests/tests/swfs/avm2/wrong_arg_count/Test.as

```diff
@@ -0,0 +1,30 @@
+﻿package {
+	public class Test {
+	}
+}
+
+import com.ruffle.MyClass;
+
+function someFunc(arg: String) {}
+
+var cls = MyClass;
+var func = someFunc;
+var method = new MyClass().myMethod;
+try {
+	new cls(true, "Bad arg");
+} catch (e) {
+	trace("Caught error: " + e);
+}
+
+try {
+	func("First", "Bad second", true);
+} catch (e) {
+	trace("Caught error: " + e);
+}
+
+
+try {
+	method("First", "Bad second", true);
+} catch (e) {
+	trace("Caught error: " + e);
+}
```

Comment:
Can you add additional tests for calling functions with too few parameters, and also for variadic functions?